wordpress-mobile/WordPress-Android
# Convert media picker activity to kotlin

This PR converts MediaPickerActivity to Kotlin in order to make future changes easier. There are no functionality changes

To test:
- Go to Media picker (add a new story)
- Check that it starts correctly
- Select items
- Make sure the items are selected correctly when you quit the activity

PR submission checklist:
